File: src/programs/java/FibonacciSeries.java
package programs.java;

public class FibonacciSeries {

    public static void main(String args[]) {
        int value=4;
        for(int i=0;i<value;i++){
            System.out.print(fibonacci(i)+" ");
        }
    }

    static int fibonacci(int value) {
        if (value <= 1) {
            return value;
        }
        return fibonacci(value - 1) + fibonacci(value - 2);
    }
}

File: src/programs/kotlin/FibonacciSeries.kt
package programs.kotlin

class FibonacciSeries {

    companion object{

        @JvmStatic
        fun main(args:Array<String>){
            val value=6
            for(i in 0 until value ){
                print("${findFibonacci(i)} ")
            }
        }

        private fun findFibonacci(value:Int):Int{
            if(value<=1)
                return value
            return findFibonacci(value-1)+ findFibonacci(value-2)
        }

    }

}